{"name": "test_get_universe_invalid_version", "status": "passed", "description": "\n        Verify that incorrect universe version is recognized.\n        ", "attachments": [{"name": "log", "source": "3abbcd2f-6363-4c4e-9125-bb3415f96e5f-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "dd46919a-660b-420d-8fe7-8641db56ae77-attachment.txt", "type": "text/plain"}], "start": 1656452178997, "stop": 1656452178999, "uuid": "4548b06d-f8e0-48d3-8f2b-f698eb22f8b9", "historyId": "05bad3f9a10030b599636cd52877b8f2", "testCaseId": "1b7506ecefcf44099049afcfc4d94415", "fullName": "im_v2.example1.universe.test.test_example1_universe.TestGetUniverse1#test_get_universe_invalid_version", "labels": [{"name": "parentSuite", "value": "im_v2.example1.universe.test"}, {"name": "suite", "value": "test_example1_universe"}, {"name": "subSuite", "value": "TestGetUniverse1"}, {"name": "host", "value": "8331b618a6a7"}, {"name": "thread", "value": "10537-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "im_v2.example1.universe.test.test_example1_universe"}]}